kitsune/search/v2/es7_utils.py

```diff
@@ -111,7 +111,7 @@ def index_object(doc_type_name, obj_id):
 
 
 @task
-def index_objects_bulk(doc_type_name, obj_ids):
+def index_objects_bulk(doc_type_name, obj_ids, timeout=10):
```

Comment:
add a var in settings

Reply:
should we remove the command line argument, then, because it could just be specified as an environment variable instead, like:
```
ES_BULK_TIMEOUT=10 ./manage.py es7_reindex
```

Reply:
Oh, actually, that won't work with celery - because the workers won't have that specified value in their environment